environment_osx.yml

```diff
@@ -10,7 +10,7 @@ channels:
 
 dependencies:
   - aiohttp
-  - cartopy <0.24  # https://github.com/ESMValGroup/ESMValTool/issues/3767
+  - cartopy
   - cdo >=2.3.0
   - cdsapi
   - cf-units
```

Comment:
Remove comment for `esmpy` below just like you did for other env?

Reply:
done in 8d1deaf33e0e910562245c8a78b3639a98f3dba2